src/utils/summarizer.py

```
@@ -136,208 +137,186 @@ async def create_long_summary(
-            should_create_long,
-            long_messages,
-            latest_long_summary,
-        ) = await should_create_summary(
-            db,
-            workspace_name,
-            session_name,
-            peer_name,
-            message_id,
-            summary_type=SummaryType.LONG,
-        )
+        async def create_long_summary():
```

Ruff (0.12.2)

Missing return type annotation for private function `create_long_summary`

Add return type annotation: `None`

(ANN202)